src/main/java/com/gmail/inverseconduit/javadoc/ClassInfo.java

```diff
@@ -1,63 +1,144 @@
 package com.gmail.inverseconduit.javadoc;
 
+import java.util.Collection;
 import java.util.List;
 
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMultimap;
+import com.google.common.collect.Multimap;
+
 /**
  * Holds the Javadoc info of a class.
  * @author Michael Angstadt
  */
 public class ClassInfo {
-	private final String fullName;
+	private final ClassName name, superClass;
 	private final String description;
-	private final String url;
 	private final List<String> modifiers;
+	private final List<ClassName> interfaces;
```

Comment:
these both could very well be Sets. that would emphasize the uniqueness of modifiers and interfaces. There is no real need to keep them ordered, is there?